### DISCUSSION TITLE
How to modify the feed url?

### DISCUSSION THREAD

**Discussion by MSicc** [cotes2020/jekyll-theme-chirpy#2023] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I have migrated from WordPress to Jekyll. My WordPress feed was /feed/, but this theme (which is really awesome!) has /feed.xml

How can I change the feed url? 

### Operations you have already tried

Searching the web for solutions, just found jekyll-feed, but couldn't get it to work with this theme

**Comment by cotes2020** [comment]
If I understand correctly, consider modifying the `permalink` to `/feed/`  in _feed.xml_:

https://github.com/cotes2020/jekyll-theme-chirpy/blob/4ef3cd8efc90518cc56b4e92306f81aa36feee70/assets/feed.xml#L3

**Reply by MSicc** [reply]
thanks for the quick response.  I don't have that file in my fork available to me. 

If I understood everything correctly (new to Jekyll), it should work if I copy the file over and modify it from there, right?

**Reply by cotes2020** [reply]
Yes. For more information see: <https://jekyllrb.com/docs/themes/#overriding-theme-defaults>.

The `/feed/` seems to be a clean path, I'll consider redirecting it to the current `/feed.xml` in the future!

**Reply by MSicc** [reply]
Awesome, will try that and report back

**Reply by MSicc** [reply]
copying the file alone wasn't enough, I needed to hardcode the full rss url for the moment in the contact.yml for the sidebar rss icon

**Reply by cotes2020** [reply]
You can use a redirect in feed.xml instead of modifying its `permalink`.

```diff
 ---
 layout: compress
 permalink: /feed.xml
+redirect_from:
+  - /feed/
 # Atom Feed, reference: https://validator.w3.org/feed/docs/atom.html
 ---
 ```
